src/main/java/org/tron/p2p/connection/ChannelManager.java

```diff
@@ -57,6 +57,7 @@ public class ChannelManager {
       .newBuilder().maximumSize(2000).build(); //ban timestamp
 
   private static boolean isInit = false;
+  public static boolean isShutdown = false;
```

Comment:
The definition prefix needs to add volatile

Reply:
ok, i will fix it.